core/src/main/java/com/klaytn/caver/transaction/AbstractTransaction.java

```diff
@@ -657,4 +652,61 @@ public String getChainId() {
     public List<KlaySignatureData> getSignatures() {
         return signatures;
     }
+
+    private void setType(String type) {
+        this.type = type;
+    }
+
+    private void setFrom(String from) {
+        //"From" field in LegacyTransaction allows null
+        if(this instanceof LegacyTransaction) {
+            if(from == null || from.isEmpty() || from.equals("0x")) from = "0x";
```

Comment:
isEmpty will return 'true' when String is ""?

Reply:
Yes. It check if String length has 0